cmd/runner/main.go

```diff
@@ -1,7 +1,72 @@
 package main
 
-import "fmt"
+import (
+	"flag"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/benchttp/runner/report"
+	"github.com/benchttp/runner/request"
+)
+
+const (
+	DefaultConcurrency = 1
+	DefaultRequests    = 0 // Use duration as exit condition if omitted.
+	DefaultDuration    = 60
+	DefaultTimeout     = 10
```

Comment:
My thought on this was: it will appear in godocs as it is public. Which can be cool,  because it provide useful information to someone who would use this.

Reply:
Hah good point! Makes sense, although I'm surprised it actually doesn't appear 🤔 It seems the docs from `cmd` are ignored anyway.